ortools/constraint_solver/routing.h

```diff
@@ -2550,12 +2550,9 @@ class RoutingDimension {
   /// never we called on any (end, _). If pre_travel_evaluator or
   /// post_travel_evaluator is -1, it will be taken as a function that always
   /// returns 0.
-  // TODO(user): Remove if !defined when routing.i is repaired.
-#if !defined(SWIGPYTHON)
```

Comment:
remove this you also allow it in .Net wrapper, not sure you have checked it...
On my way to do some tests...

Reply:
My bad this is the contrary, it unignores it in Python contrary to previous behaviour and it make examples failing...